src/main/main.cpp

```diff
@@ -184,6 +184,9 @@ usage(int err = 1)
           "                           (Default is STELLAR_NETWORK_ID "
           "environment variable)\n"
           "      --test               Run self-tests\n"
+          "           --all-versions  Run with all possible protocol versions\n"
```

Comment:
I don't think we need to have `--all-versions` or `--version` documented here. They will appear in `--test -?` now. The documentation added here should be specified where they are added to the CLI.

Reply:
ah yes the new version of Clara makes this irrelevant